setup.py

```diff
@@ -53,19 +62,26 @@ def run_tests(self):
     "pytimber>=2.8.0",
 ]
 
-# Dependencies that should only be installed for test purposes
-TEST_DEPENDENCIES = [
-    "pytest>=5.2",
-    "pytest-cov>=2.6",
-    "hypothesis>=3.23.0",
-    "attrs>=19.2.0",
-]
-
-# pytest-runner to be able to run pytest via setuptools
-SETUP_REQUIRES = ["pytest-runner"]
-
-# Extra dependencies for tools
-EXTRA_DEPENDENCIES = {"doc": ["sphinx", "travis-sphinx", "sphinx_rtd_theme"]}
+# Extra dependencies
+EXTRA_DEPENDENCIES = {
+    "setup": [
+        "pytest-runner"
+    ],
+    "test": [
+        "pytest>=5.2",
+        "pytest-cov>=2.6",
+        "hypothesis>=3.23.0",
+        "attrs>=19.2.0",
+    ],
+    "doc": [
+        "sphinx",
+        "travis-sphinx",
+        "sphinx_rtd_theme"
+    ],
+}
+EXTRA_DEPENDENCIES.update(
+    {'all': [elem for list_ in EXTRA_DEPENDENCIES.values() for elem in list_]}
```

Comment:
Some verbosity can't hurt. Maybe `[requirement for extra_reqs in EXTRA_DEPENDENCIES.values() for requirement in extra_reqs]`?

Reply:
I don't think it makes it any more readable and just makes the line much longer.